pkg/mover/doc_load_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package mover
+
+import (
+	"fmt"
+)
+
+// Load a chart and its dependencies from a local intermediate bundle tarball
+// into a registry and or repository
+func Example_load() {
+	// i.e ./mariadb-7.5.relocated.tgz
+	destinationPath := "%s-%s.relocated.tgz"
+
+	// Initialize the Mover action
+	chartMover, err := NewChartMover(
+		&ChartMoveRequest{
+			Source: Source{
+				Chart: ChartSpec{
+					// The source intermediate bundle path to place the charts and all its dependencies
```

Comment:
> // The source intermediate bundle path to place the charts and all its dependencies

the intermediate source tarball where the source chart and container images are present.

I'd also mention to see the Save example.